apps/ai-content-generator/.eslintrc.json

```diff
@@ -0,0 +1,27 @@
+{
+  "env": {
+    "browser": true,
+    "es2021": true
+  },
+  "extends": [
+    "eslint:recommended",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:react/recommended",
+    "plugin:react/jsx-runtime",
+    "plugin:react-hooks/recommended"
+  ],
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "ecmaVersion": "latest",
+    "sourceType": "module"
+  },
+  "plugins": ["@typescript-eslint", "react"],
+  "rules": {
+    "react/no-unknown-property": ["error", { "ignore": ["css"] }]
```

Comment:
This prevents the error `Unknown property 'css' found`, see more info here: https://emotion.sh/docs/eslint-plugin-react